vm/src/stdlib/itertools.rs

```diff
@@ -1612,12 +1689,17 @@ mod decl {
     #[pyclass(name = "permutations")]
     #[derive(Debug, PyPayload)]
     struct PyItertoolsPermutations {
-        pool: Vec<PyObjectRef>,               // Collected input iterable
-        indices: PyRwLock<Vec<usize>>,        // One index per element in pool
-        cycles: PyRwLock<Vec<usize>>,         // One rollover counter per element in the result
-        result: PyRwLock<Option<Vec<usize>>>, // Indexes of the most recently returned result
-        r: AtomicCell<usize>,                 // Size of result tuple
-        exhausted: AtomicCell<bool>,          // Set when the iterator is exhausted
+        pool: Vec<PyObjectRef>,
+        // Collected input iterable
```

Comment:
The comment place here is looking weird. Isn't this expected to be put on upper line?

Reply:
Hi,

Thanks for response,

Yeah, actually I am not sure how this is getting formatted (maybe because of my IDE).
Anyway will fix it and push this.

Now Ran (cargo fmt -all)

Reply:
some tests are failing after this formatting, Is it possible to run tests again?